GOVERNANCE.md

```diff
@@ -0,0 +1,191 @@
+
+* Responsibilities include:
+    * All responsibilities of a maintainer
+    * Determining strategy and policy for the project
+    * Participating in, and leading, community meetings
+* Requirements
+    * Experience as a Maintainer for at least 3 months
+    * Demonstrates a broad knowledge of all components, repositories, and subprojects of the Podman project.
+    * Is able to exercise judgment for the good of the project, independent of their employer, friends, or team
+    * Mentors new Maintainers and Core Maintainers
+    * Able to make decisions and contributions affecting the whole project, including multiple subprojects and repositories
+    * Can commit to maintaining a high level of contribution to the project as a whole
+* Additional privileges:
+    * Represent the project in public as a senior project member
+    * Communicate with the CNCF on behalf of the project
+    * Have a vote in Maintainer decision-making meetings
+
+#### Process of becoming a Core Maintainer Member:
+1. A current maintainer must be sponsored by Core Maintainer. The Core Maintainer will open a PR against the main Podman repository and add the nominee as a Core Maintainer in the MAINTAINERS.md file.
+2. The nominee will add a comment to the PR testifying that they agree to all requirements and responsibilities of becoming a Core Maintainer.
+3. A majority of the current Core Maintainers must then approve the PR.
+4. If, for some reason, all existing members are inactive according to the Inactivity policy below or there are no Core Maintainers due to resignations, a supermajority vote of maintainers can bypass this process and approve new Core Maintainers directly.
+
+### Emeritus Maintainer
+Emeritus Maintainers are former Maintainers or Core Maintainers whose status has lapsed, either voluntarily or through inactivity. We recognize that these former maintainers still have valuable experience and insights, and maintain Emeritus status as a way of recognizing this. Emeritus Maintainer also offers a fast-tracked path to becoming a Maintainer again, should the contributor wish to return to the project.
+
+Emeritus Maintainers have no responsibilities or requirements beyond those of an ordinary Contributor.
+
+#### Process of becoming an Emeritus Maintainer:
+1. A current Maintainer or Core Maintainer may voluntarily resign from their position by making a pull request changing their role in the OWNERS file. They may choose to remove themselves entirely or to change their role to Emeritus Maintainer.
+2. Maintainers and Core Maintainers removed due to the Inactivity policy below may be moved to Emeritus Status.
+
+## Maintainers File
```

Comment:
should this PR introduce that file?

Reply:
It should, yes. I'll do that tomorrow.
I'll also start up a reviewers guide for PRs.